Makefile

```diff
@@ -54,3 +55,10 @@ black-apply: # Apply changes with 'black'
 
 ruff-apply: # Resolve 'fixable errors' with 'ruff'
 	pipenv run ruff check --fix .
+
+# Development commands
```

Comment:
I'd propose updating to something like "Minio local S3 commands"

Reply:
Updated to: 

```text
####################################
# MinIO local S3 commands
####################################
```